print-ospf.c

```diff
@@ -611,6 +654,226 @@ ospf_print_tos_metrics(netdissect_options *ndo,
     }
 }
 
+static int
+ospf_print_ri_lsa_sid_label_range_tlv(netdissect_options *ndo, const uint8_t *tptr, u_int tlv_length)
+{
+    u_int subtlv_type, subtlv_length;
+
+    while (tlv_length >= 4) {
+
+	ND_TCHECK_4(tptr);
+	subtlv_type = GET_BE_U_2(tptr);
+	subtlv_length = GET_BE_U_2(tptr+2);
+	tptr+=4;
+	tlv_length-=4;
+
+	/* Infinite loop protection. */
+	if (subtlv_type == 0 || subtlv_length == 0) {
```

Comment:
nd_print_invalid(ndo); before the return?

Reply:
ok